pkg/agent/controller/traceflow/traceflow_controller.go

```diff
@@ -345,7 +347,7 @@ func (c *Controller) startTraceflow(tf *crdv1alpha1.Traceflow) error {
 	tfState := traceflowState{
 		name: tf.Name, tag: tf.Status.DataplaneTag,
 		liveTraffic: liveTraffic, droppedOnly: tf.Spec.DroppedOnly && liveTraffic,
-		receiverOnly: receiverOnly, isSender: isSender}
+		receiverOnly: receiverOnly, isSender: isSender, packet: packet}
```

Comment:
I didn't find who consumes the packet field, could you help figure out?

Reply:
Thanks. This is debug code used previously and the consumer is already removed.
Fixed.